Source/CheatSheet/CQLBasics-cheatsheet.md

```diff
@@ -0,0 +1,121 @@
+# CQL Basics Cheat Sheet
```

Comment:
General comment: I would suggest we add some links to the CQL spec in each of the sections.

Reply:
Applied